src/ngx_http_lua_socket_tcp.c

```diff
@@ -1002,6 +1004,46 @@ ngx_http_lua_socket_resolve_handler(ngx_resolver_ctx_t *ctx)
 }
 
 
+static void
+ngx_http_lua_socket_init_peer_connection_addr_text(ngx_peer_connection_t *pc)
```

Comment:
We do not have an existing nginx C function to call directly? ;)

Reply:
Yes. It seems that Nginx only initialize `c->addr_text` in `ngx_event_accept`:
https://github.com/nginx/nginx/blob/41d8ea8c8d268555d3cbd8dd2ab32dcc06658209/src/event/ngx_event_accept.c#L269

Reply:
@spacewander Yeah, I saw that.